modules/claims_api/app/models/claims_api/evss_claim.rb

```diff
@@ -56,5 +56,44 @@ def supporting_documents
         []
       end
     end
+
+    def self.services_are_healthy?
+      # TODO: we should add check for Okta and SAML Proxies being up as well
```

Comment:
This comment seems to lead us toward an unsustainable future because neither Okta nor the SAML proxy are direct dependencies of the API. They are definitely related, because you need an access token, which would come from interactions with both of those services. However, if we have every API concerned with auxillary dependencies, notifications will become a tangled mess. The practical impact of a notification would be less clear. For example, the API could be reported as out of service but still be able to serve requests carrying an access token that was generated before either component went down, so long as the access token had already been verified before okta went down.

Reply:
@dvogel You are correct that once an initial token is created, we may not need to communicate with the SAML proxy or Okta, unless a refresh token is being used and it is time for a refresh. The reason we added this comment is to be able to mark our services as down (and potentially degraded on the status page) when a service we depend on is down. However it brings up another question on whether the SAML proxy will respond gracefully when Okta cannot be reached. @dvogel do you know how that would be handled?

Reply:
@charleystran the SAML proxy will still be reachable if Okta cannot be reached, but the OAuth proxy will not

Reply:
@mjlumetta I am not 100% sure what you are voting for in this case. Our position was that since the saml proxy and okta are dependencies of V1 of the claims API, we should add them into our overall healthchecks. We would do this separately as 2 different checks.

Reply:
no position on that! i was trying to answer your question about whether the SAML proxy will respond gracefully if Okta is down (yes). but i'm adding the clarification that the OAuth proxy, which lives in the same repo as the SAML proxy, will not be up if Okta is down. does that help?

Reply:
@mjlumetta what does gracefully mean in your words? I since its hard to force the error, I am looking to see what kind of response we can expect if Okta is down. If Okta is down, what is the response code we would receive?

Reply:
yeah, so... not sure what endpoint you're going to use as a healthcheck or what request you'll make. but the SAML proxy doesn't depend on Okta in and of itself. it does **redirect** to Okta, so i think you would want to test for a 302 rather than a 200 if you're going through the flow. or if there is a health check endpoint already that i don't know about/forgot about that can return 200, i would assume that would be unaffected.

Reply:
these were moved out of the model and actually the check was moved out of the individual controllers since both v0 and v1 provide the same checks as of now, there was no need to duplicate the logic.